Keyboards/KeyboardsBase/KeyboardViewController.swift

```diff
@@ -212,6 +217,68 @@ class KeyboardViewController: UIInputViewController {
     leftAutoPartition.backgroundColor = .clear
     rightAutoPartition.backgroundColor = .clear
   }
+  
+  
+  // Logic to create tooltip
+  func createInformationStateDatasource(text: String, backgroundColor: UIColor) -> ToolTipViewDatasource {
+    let theme = ToolTipViewTheme(backgroundColor: backgroundColor, textFont: nil, textColor: UIColor.black, textAlignment: .center, cornerRadius: 10, masksToBounds: true)
+    let content = getENTooltipContent(content: text, fontSize: 10)
+
+    return ToolTipViewDatasource(content: content, theme: theme)
+  }
+  
+  func setInformationState() {
+    let contentData: [String] = InformationToolTipData.getContent()
+    let backgroundColors: [UIColor] = [#colorLiteral(red: 0.6392156863, green: 0.7764705882, blue: 0.1921568627, alpha: 1), #colorLiteral(red: 0.6274509804, green: 0.7333333333, blue: 0.9098039216, alpha: 1),  #colorLiteral(red: 0.9725490196, green: 0.8666666667, blue: 0.3960784314, alpha: 1)]
```

Comment:
I like the initiative of adding some color into something that's pretty bland. If we are gonna do colors for this, then they at the very least should be Scribe or [Wikidata colors](https://www.schemecolor.com/wikidata-logo-colors.php) 🤔 Then again, adding the Wikidata colors might be problematic based on light and dark modes for Scribe (I think they're too dark for dark mode and will blend in), so I'm thinking having this be the normal key background color and the key text color might be best so we don't need to worry about color modes.